test/kuttl/tests/glance_disk_formats/01-create_image.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+    - command: |
+        oc -n $NAMESPACE exec -it openstackclient --stdin=false -- /bin/sh -c "DOMAIN=glance-default-single.$NAMESPACE.svc ./scripts/create-image.sh raw"
```

Comment:
Looking at the script I think you don't need `"DOMAIN=glance-default-single.$NAMESPACE.svc`.
Remove it from L5 and L7.